### ISSUE TITLE
Implement unified 32 and 64 bit projection interface

### ISSUE THREAD

**Issue by Pessimistress** [uber/deck.gl#1492]
#### RFC
https://github.com/uber/deck.gl/blob/master/dev-docs/RFCs/vNext/unified-32-and-64-bit-project-api-rfc.md

#### To Do List
  - [x] Shader modules
  - [x] Update core layers
  - [x] Update experimental layers . -> pending on core layer release on v5.2, add a separate task post v5.2
  - [x] Doc update
  - [x] Whats new update
  - [x] Render test
